proxy/pom.xml

```diff
@@ -87,16 +87,6 @@
       <version>3.1</version>
     </dependency>
     <dependency>
-      <groupId>com.squareup</groupId>
```

Comment:
The "tape" dependency was simply a duplicate in the proxy's pom.xml, and two "caffeine" dependencies were referring to different version, so maven was complaining about both of them.